add-code-context.py

```
@@ -6,91 +6,255 @@
+        if not path.exists():
+            print(f"❌ Error: File not found: {file_path}")
+            return None
+            
+        with open(path, 'r') as f:
+            data = json.load(f)
+            
+        # Handle both array and object formats
+        if isinstance(data, list):
+            data = {'issues': data}
+        elif not isinstance(data, dict) or 'issues' not in data:
+            print(f"❌ Error: Invalid JSON format. Expected 'issues' array.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)